src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -334,37 +378,57 @@ namespace {
         */
 
         auto restart_skip = currentStep < this->m_timeMap.restart_offset();
+        ScheduleLogger logger(*this, restart_skip);
+        {
+            const auto& schedule_keyword = section.getKeyword<ParserKeywords::SCHEDULE>();
+            const auto& location = schedule_keyword.location();
+            current_file = location.filename;
+            logger(fmt::format("Processing dynamic information from\n{} line {}\n", current_file, location.lineno));
+            logger(fmt::format("Initializing report step {}/{} at {} {} days line {}",
```

Comment:
The `days` here is technically wrong for LAB units.  How much work would it be to output the unit-system dependent time unit name instead?

Reply:
IT is ONLY the string days which is wrong. I'll fix